# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1240630321 25200
#      Fri Apr 24 20:32:01 2009 -0700
# Node ID 761fc0d3d980a3cc0e3a5859726ab2c0a6a26c39
# Parent  5247e89688e13821d971b3c42a11af97b4c75114
Update section 17.2 (Complex Arithmetic) of arith.txi

Use Tex in more of the doc strings for pretty printing in pdf format.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2009-04-24  Rik  <rdrider0-list@yahoo.com>
+	* interpreter/arith.txi: Update section 17.2 (Complex Arithmetic) of
+	arith.txi
+	Use Tex in more of the doc strings for pretty printing in pdf format.
+
 2009-04-24  Rik  <rdrider0-list@yahoo.com>
 	* interpreter/octave.pdf: 
 	Eliminate most overfull errors when running texi2pdf for generating 
 	pdf documentation.
 	Use @smallexample when necessary to reduce font for long lines.
 	Reword variables or phrases so that Tex can break them at a better spot.
 
 2009-04-21  Rik  <rdrider0-list@yahoo.com>
diff --git a/doc/interpreter/arith.txi b/doc/interpreter/arith.txi
--- a/doc/interpreter/arith.txi
+++ b/doc/interpreter/arith.txi
@@ -71,20 +71,17 @@ sin ([1, 2; 3, 4])
 
 @DOCSTRING(realsqrt)
 
 @DOCSTRING(sqrt)
 
 @node Complex Arithmetic
 @section Complex Arithmetic
 
-The following functions are available for working with complex
-numbers.  Each expects a single argument.  Given a matrix they work on
-an element by element basis.  In the descriptions of the following
-functions,
+In the descriptions of the following functions,
 @iftex
 @tex
 $z$ is the complex number $x + iy$, where $i$ is defined as
 $\sqrt{-1}$.
 @end tex
 @end iftex
 @ifinfo
 @var{z} is the complex number @var{x} + @var{i}@var{y}, where @var{i} is
diff --git a/scripts/general/cplxpair.m b/scripts/general/cplxpair.m
--- a/scripts/general/cplxpair.m
+++ b/scripts/general/cplxpair.m
@@ -21,26 +21,26 @@
 ## @deftypefnx {Function File} {} cplxpair (@var{z}, @var{tol})
 ## @deftypefnx {Function File} {} cplxpair (@var{z}, @var{tol}, @var{dim})
 ## Sort the numbers @var{z} into complex conjugate pairs ordered by 
 ## increasing real part.  Place the negative imaginary complex number
 ## first within each pair.  Place all the real numbers (those with
 ## @code{abs (imag (@var{z}) / @var{z}) < @var{tol})}) after the
 ## complex pairs.
 ##
-## If @var{tol} is unspecified the default value is 100*@var{eps}.
+## If @var{tol} is unspecified the default value is 100*@code{eps}.
 ##
 ## By default the complex pairs are sorted along the first non-singleton
 ## dimension of @var{z}.  If @var{dim} is specified, then the complex
 ## pairs are sorted along this dimension.
 ##
-## Signal an error if some complex numbers could not be paired.  Requires
-## all complex numbers to be exact conjugates within @var{tol}, or signals an 
-## error.  Note that there are no guarantees on the order of the returned
-## pairs with identical real parts but differing imaginary parts.
+## Signal an error if some complex numbers could not be paired.  Signal an
+## error if all complex numbers are not exact conjugates (to within
+## @var{tol}).  Note that there is no defined order for pairs with identical
+## real parts but differing imaginary parts.
 ##
 ## @c Set example in small font to prevent overfull line
 ## @smallexample
 ## cplxpair (exp(2i*pi*[0:4]'/5)) == exp(2i*pi*[3; 2; 4; 1; 0]/5)
 ## @end smallexample
 ## @end deftypefn
 
 ## FIXME: subsort returned pairs by imaginary magnitude
diff --git a/src/mappers.cc b/src/mappers.cc
--- a/src/mappers.cc
+++ b/src/mappers.cc
@@ -169,19 +169,26 @@ See arg.\n\
 
   return retval;
 }
 
 DEFUN (arg, args, ,
     "-*- texinfo -*-\n\
 @deftypefn {Mapping Function} {} arg (@var{z})\n\
 @deftypefnx {Mapping Function} {} angle (@var{z})\n\
-Compute the argument of @var{z}, defined as\n\
-@var{theta} = @code{atan2 (@var{y}, @var{x})}.\n\
-in radians.  \n\
+Compute the argument of @var{z}, defined as,\n\
+@iftex\n\
+@tex\n\
+$\\theta = atan2 (y, x),$\n\
+@end tex\n\
+@end iftex\n\
+@ifnottex\n\
+@var{theta} = @code{atan2 (@var{y}, @var{x})},\n\
+@end ifnottex\n\
+in radians.\n\
 \n\
 For example,\n\
 \n\
 @example\n\
 @group\n\
 arg (3 + 4i)\n\
      @result{} 0.92730\n\
 @end group\n\
